politeiawww/api/cms/v1/api.md

```diff
@@ -1765,6 +1765,179 @@ Reply:
 }
 ```
 
+### `Proposal Billing Summary`
+
+Retrieve all billing information for all approved proposals.
+
+This retrieves the tokens for approved proposals and uses those tokens to
+search through the database for invoices that have line-items that have that
+as proposal token added.
+
+Note: This call requires admin privileges.
+
+**Route:** `GET /v1/admin/spendingsummary`
```

Comment:
It's documented as `/admin/spendingsummary` here but the route is `/proposals/spendingsummary`